binding.gyp

```diff
@@ -74,6 +74,17 @@
           },
         }]
       ]
+    },
+    {
+      "target_name": "action_after_build",
+      "type": "none",
+      "dependencies": [ "<(module_name)" ],
+      "copies": [
+          {
+            "files": [ "<(PRODUCT_DIR)/<(module_name).node" ],
```

Comment:
I believe those other files are for debugging purposes.

They probably aren't necessary, but if you can find a way to include them that would be good.